pwndbg/commands/version.py

```diff
@@ -5,8 +5,10 @@
 
 import argparse
 import os
+import re
 import sys
 from platform import platform
+from platform import system
```

Comment:
nitpick: I think we should just `import platform` and then call `platform.platform()` and `plaform.system()`. People generally think of `os.system()` when they see system, so the code below will be more clear like this.